src/client/features/view/components/tooltips/generateContent.js

```diff
@@ -0,0 +1,238 @@
+const h = require('hyperscript');
+const config = require('../../config');
+
+
+//Handle name related metadata fields
+const standardNameHandler = (pair) => makeTooltipItem(pair[1], 'Approved Name: ');
+const displayNameHandler = (pair) => makeTooltipItem(pair[1], 'Display Name: ');
+const nameHandler = (pair, trim) => {
```

Comment:
The trim parameter is giving me a hunch that this function is doing too much.

Reply:
Removed trim parameter from all handler functions and created separate functions to handle cases involving trim